CHANGELOG.md

```diff
@@ -38,6 +38,34 @@ All notable changes to this project will be documented in this file.
 - `0.1.x` Releases - [0.1.0](#010)
 
 ---
+
+## Unreleased
+
+#### Added
+
+#### Updated
+
+#### Deprecated
+
+#### Removed
+
+#### Fixed
+
+---
+
+## [4.1.1](https://github.com/Nike-Inc/SQift/releases/tag/4.1.1)
+
+Release on 2019-05-02. All issues associated with this milestone can be found using this
+[filter](https://github.com/Nike-Inc/SQift/milestone/11?closed=1).
+
+#### Fixed
+-  Issue where Xcode project was incorrectly set to Swift 5.0 instead of 4.2 breaking Carthage and submodule support.
+  - Updated by [William Thompson](https://github.com/eSpecialized) in Pull Request [#25](https://github.com/Nike-Inc/SQift/pull/25).
```

Comment:
`Updated` -> `Fixed` and then let's do fixed by Bill and Greg.

Reply:
Changed